src/SceneView.tsx

```diff
@@ -102,7 +103,13 @@ export default class SceneView<T extends Route> extends React.Component<
   };
 
   render() {
-    const { navigationState, index, layout, style } = this.props;
+    const {
+      navigationState,
+      index,
+      layout,
+      containerStyle,
+      style: defaultStyle,
```

Comment:
Where is this `style` prop coming from? It was removed in only place which passed it in the PR

Reply:
After digging around, it comes from the class component itself. Regardless if we pass in our own prop called `style` into this component, it will have AT LEAST these styles (before my changes)
```
{"style": [undefined, {"bottom": 0, "height": undefined, "left": 0, "position": "absolute", "right": 0, "top": 0, "width": undefined}]}
```

where it says `undefined` as the first array item, that's where our passed in `style` prop would be

And the styles above gets merged into whatever we pass in to `style` prop, any style property that is `bottom/height/left/position/right/top/width` will always get overridden by the 
`{"bottom": 0, "height": undefined, "left": 0, "position": "absolute", "right": 0, "top": 0, "width": undefined}]}` styles

Reply:
> 
> 
> Where is this `style` prop coming from? It was removed in only place which passed it in the PR

I don't completely understand the (quite complicated) structure of the `TabView` component, but as far as I can tell its coming from `Pager` aka. [`PagerViewAdapter` here](https://github.com/satya164/react-native-tab-view/blob/97c0857b534d08c12c06cb0c1b348a16c3ac8da9/src/PagerViewAdapter.tsx#L126), which wraps `SceneView`.

`sceneContainerStyle` is passed as the `style` prop to `SceneView` but somehow overwritten by the `style` prop from `PagerViewAdapter`. This causes `sceneContainerStyle` to not be applied at all, renaming the `style` prop from `SceneView` to anything else (could be `containerStyle` like here or even just `style2`) fixes that.